src/main/java/Todo.java

```diff
@@ -0,0 +1,10 @@
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+
+    //override to string in task
```

Comment:
Perhaps you could consider a following a single coding standard for all the comments you've put ? Eg: Leaving a space at the beginning and capitalizing the first letter of every comment